src/Card.svelte

```diff
@@ -89,6 +89,11 @@ limitations under the License.
             el.style.setProperty('--ha-card-backdrop-filter', 'none');
         }
 
+        // eslint-disable-next-line svelte/no-dom-manipulating
+        container.replaceWith(el);
```

Comment:
This change prevents a console error by replacing container before calling resizeObserver which may cause a tick to take place and container elemeny may have been removed but svelte not fully updated container binding.